Cargo.toml

```diff
@@ -11,7 +11,7 @@ rust-version = "1.81.0"
 
 [package.metadata.docs.rs]
 features = ["rt"]
-targets = ["thumbv8m.main-none-eabihf", "riscv32imac-unknown-none-elf"]
+targets = ["thumbv8m.main-none-eabi", "thumbv8m.main-none-eabihf", "riscv32imac-unknown-none-elf"]
```

Comment:
Just an observation: rp235x-hal only uses `targets = ["thumbv8m.main-none-eabihf"]` in `[package.metadata.docs.rs]`.
As far as I understand it, listing all the supported targets is correct. Should we update rp235x-hal as well?